experiments/redis/src/main/scala/org/apache/gearpump/streaming/redis/RedisMessage.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.streaming.redis
+
+import java.nio.charset.Charset
+
+object RedisMessage {
+
+  private def toBytes(string: String,
+                      charset: Charset = Charset.forName("UTF8")
+                     ): Array[Byte] = string.getBytes(charset)
+
+  case class PublishMessage(message: Array[Byte]) {
+    def this(message: String) = this(toBytes(message))
+  }
+
+  case class SetMessage(key: Array[Byte], value: Array[Byte]) {
+    def this(key: String, value: String) = this(toBytes(key), toBytes(value))
+  }
+
+  case class LPushMessage(key: Array[Byte], value: Array[Byte]) {
```

Comment:
It's unclear how this message type can be used. `LPushMessage` looks the same as `RPushMessage`